gui/default/syncthing/core/logViewerModalView.html

```diff
@@ -29,6 +29,13 @@
     </div>
   </div>
   <div class="modal-footer">
+    <div>
+      <input type="text" id="logSearchInput" class="form-control" style="margin-bottom: 10px;" ng-keyup="logging.filter_input($event)">
+      <div>
+          <input type="checkbox" ng-click="logging.isCaseSensitive = !logging.isCaseSensitive"><span>Case Sensitive</span>
```

Comment:
Does anyone ever want case-sensitive matching, in a quick view like this (as opposed to some sort of forensic log grepping or whatever)? I think we could simplify this away.